src/main/java/org/apache/commons/io/input/BrokenInputStream.java

```diff
@@ -117,10 +117,10 @@ public int read() throws IOException {
     /**
      * Throws the configured exception.
      *
-     * @throws IOException always thrown
+     * @throws IOException always throws the exception configured in the constructor.
```

Comment:
Since there is more than one constructor, I think we should say "in a constructor", not in "in the...".

Reply:
As you like - I read this as 'in the constructor [you callled]', but I don't have a strong feeling either way.

Reply:
How about "throws the exception configured on construction"?

Reply:
> How about "always throws the exception configured on construction"?

I'm not going to go round and round on such a small point. I'll likely review post merge and resolve whatever is left.